manifests/repo.pp

```diff
@@ -19,6 +19,8 @@
                 key          => $key_id,
                 key_source   => $gpgkey,
                 pin          => $pin,
+                before       => Anchor['cassandra::repo::end'],
```

Comment:
Can't we write :  
```
before => Package['dsc']
```
to force the source list to be updated before the package installation ?